packages/office-addin-debugging/src/commands.ts

```diff
@@ -56,6 +56,7 @@ export async function start(manifestPath: string, platform: string | undefined,
         const sourceBundleUrlComponents = new devSettings.SourceBundleUrlComponents(
             command.sourceBundleUrlHost, command.sourceBundleUrlPort,
             command.sourceBundleUrlPath, command.sourceBundleUrlExtension);
+        const webview = parseWebview(command.webview) | devSettings.WebViewType.Edge;
```

Comment:
add the type onto the variable: `const webview: WebViewType | undefined = parseWebView()`